src/styles/main.scss

```
@@ -1,3 +1,67 @@
-body {
-  margin: 0;
+@import "./reset.scss";
+@import "./variables.scss";
+
+html {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
```

```suggestion
html {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 30px;
}
```
dont set anything except font and css vars to `html`